src/test/java/io/vertx/ext/apex/handler/SockJSHandlerTest.java

```diff
@@ -88,12 +88,10 @@ private void testGreeting(String uri) {
 
   @Test
   public void testNotFound() {
-    waitFor(7);
+    waitFor(5);
 
     testNotFound("/echo/a");
     testNotFound("/echo/a.html");
-    testNotFound("/echo//");
```

Comment:
I think these cases can be moved to the testGreeting method as they should now pass.